packages/salesforcedx-vscode-core/src/orgBrowser/metadataCmp.ts

```diff
@@ -104,4 +105,11 @@ export class ComponentUtils {
     }
     return componentsList;
   }
+
+  public async retrieveComponent(componentNode: BrowserNode) {
+    const type = componentNode.metadataType;
+    const component = componentNode.label; // this will use componentNode.fullName once folder changes are merged
```

Comment:
We'll use the 'fullName' property of the nodes once the folder changes are merged. To retrieve components from folders, we have to specify the folder that the component belongs to. We won't need to do anything special to handle that at this point, because the `fullName` property will contain the information needed ie. 'folderOne/component'.